contributor/03-contributor-ethos.md

```diff
@@ -0,0 +1,64 @@
+# Draft comments
+
+ Outline for this section
+
+     Contributors by definition work in teams that they are not familiar with. This section explains how to avoid common pitfalls when crossing team boundaries.
+
+ ## Article
+
+In the last segment we have motivated why you would want to reuse components and become active as a Contributor. This artice is sharing best practices on how to successfully contribute your changes to the host team's code base.
+
+ ### Contributor Ethos
+
+An InnerSource Contributor trying to make a contribution to the host team essentially is a guest in their home. In general a good guest is expected to behave in a certain way:
+
+* They knock at the door.
+* They anticipate and follow house rules.
+* They understand they are not the home owner and acty accordingly.
+
+How do these expectations apply to InnerSource projects?
+
+#### Getting in
+
+In the real world their are several ways that guests are invited in:
+
+* An "open" sign lets them know they can walk right in without knocking.
+* Knock on the door unexpected.
+* Knock on the door only when invited.
+
+With InnerSource projects there are several ways that code gets in:
+
+Sometimes you just open a pull request. At times you may need to get a feature request accepted before opening the pull request that fulfills it.
+
+In some organisations you may need to talk to the host team in person in addition to interacting with the project digitally. 
+
+In the same way that real-life hosts don't always appreciate a stranger knocking on their door, some InnerSource hosts appreciate some heads-up before seeing a pull request gets opened to their repo. Typically there will be an issue tracker where new features are discussed before being submitted as a pull request. A useful alternative are archived, searchable, linkable but unstructured discussion platforms: Thank of mailing lists, chat plattforms or online fora.
+
+In all of these interactions, be prepared to "sell" your contribution to the host team. Articulate the value that the contribution will bring to their ecosystem.
+
+The host team will be the one taking over maintenance for your changes. It makes sense to offer to fulfill a 30-day warranty on your submission. This can alleviate the host teams' fear of the contributors not being available for support with fixing bugs after the time on contribution.
+
+Many InnerSource projects outline how they like to be approached by potential Contributors in their README.md or CONTRIBUTING.md files. Following those recommendations greatly increases your chances of getting your contribution accepted.
+
+#### Anticipate and follow house rules
+
+The README.md or CONTRIBUTING.md often not only contains information about how to approach the host team, but also guidelines throughout the contribution process. Be sure to understand and follow them.
```

Comment:
I would like to make a greater distinction between README.md and CONTRIBUTING.md. I suggest we suggest (:-)) to use the README.md to explain what the project is about and how to USE, Build and get started with it. In contrast, the CONTRIBUTING.md  should be used to describe HOW TO CONTRIBUTE to the project (process wise, code styles, test expectations etc).  Maybe I should open an issue for review on docs in general?

Reply:
I think it makes sense to be a bit more specific in this paragraph, however I wouldn't strictly forbid contributing related docs in the README. More like: If you don't find contributing related documentation in the README they have been split out into CONTRIBUTING.md

The reasoning behind that: For small enough projects both sides of the documentation can be short enough to be kept in one document. Even if they are split, the contributor will still have to read the README anyway to first understand how the project works and can be used before even thinking about contributing.

Makes sense?

Reply:
I agree with both opinions. 
To me it appears that the shared goal is to prevent too long documentation and enable easier absorption of the knowledge contained in the documents.

With larger projects separation is totally great, with the smaller ones, combining them is fine too. What we could provide is a guidance when to separate such documents and how to arrive at a (set of) document(s) that serve their purpose well. 
I was working at such documents a while ago, let's see what became of them.